src/app/components/Common/Player/Player.jsx

```diff
@@ -58,32 +46,32 @@ class Player extends React.Component {
     Mousetrap.bind('space', togglePlayback, 'keyup');
   }
 
-  handlePrevious() {
+  handlePrevious = () => {
     const { player } = this.props.mk.instance;
 
     if (player.currentPlaybackTime < 2) {
       player.skipToPreviousItem();
     } else {
       player.seekToTime(0);
     }
-  }
+  };
 
-  handleNext() {
+  handleNext = () => {
     const { player } = this.props.mk.instance;
     player.skipToNextItem();
 
     if (player.repeatMode === 1) {
       player.seekToTime(0);
     }
-  }
+  };
 
-  // eslint-disable-next-line class-methods-use-this
-  handleAddToLibrary() {
-    // console.log('Add to library');
-    // this.props.mk.instance.addToLibrary();
-  }
+  // // eslint-disable-next-line class-methods-use-this
```

Comment:
Lets remove it completely?

Reply:
Yeah, could do. I'm assuming we will add the method back soon so didn't remove it as a reminder.

Reply:
Hum, fair enough, but I think that we will remember nonetheless

Reply:
Yeah, that's fair. We have an issue tracking it. I'll remove it in this PR, give me a second and I'll amend the commit.